cyclus_input/predicting_the_past.xml

```diff
@@ -13,7 +13,7 @@
     <spec><lib>cycamore</lib><name>Source</name></spec>
     <spec><lib>cycamore</lib><name>Storage</name></spec>
   </archetypes>
-  <xi:include href="./cyclus_input/recipes/lifetimes.xml#xpointer(/simulation/facilities/child::*)" />
+  <xi:include href="./recipes/lifetimes.xml#xpointer(/simulation/facilities/child::*)" />
```

Comment:
The lifetimes.xml file just has fragments of nodes for decaring <facilities> and <DeployInsts>, which go under two separate elements within cyclus input.

I did not try that but I will with the reactors files to see how that goes.
Thank you!